dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -17,11 +17,13 @@ KEYWORDS="~amd64 ~x86"
 IUSE="doc elogind systemd test"
 REQUIRED_USE="
 	${PYTHON_REQUIRED_USE}
-	^^ ( systemd elogind )
+	^^ ( elogind systemd )
 "
 RESTRICT="!test? ( test )"
 
 BDEPEND="
+	${PYTHON_DEPS}
+	dev-lang/swig
 	virtual/pkgconfig
 	doc? ( app-doc/doxygen )
 	test? ( dev-util/valgrind )
```

Comment:
For better accuracy in case of crosscompilation, this should be:
```
	test? (
		dev-libs/gobject-introspection
		dev-util/valgrind
		$(python_gen_cond_dep 'dev-python/pygobject:3[${PYTHON_MULTI_USEDEP}]')
		$(python_gen_cond_dep 'dev-python/python-evdev[${PYTHON_MULTI_USEDEP}]')
	)
```
https://github.com/libratbag/libratbag/blob/1097b1ba686e6a0c62868737bc767b34a039b110/meson.build#L588-L596 creates `ratbagctl.test` from `tools/ratbagctl.test.in`.
https://github.com/libratbag/libratbag/blob/f82c6f72b692af29b1f220b2e04061eec9202811/tools/ratbagctl.test.in#L32 imports `toolbox.py` created in https://github.com/libratbag/libratbag/blob/1097b1ba686e6a0c62868737bc767b34a039b110/meson.build#L581-L583 from `tools/toolbox.py`.
And https://github.com/libratbag/libratbag/blob/f82c6f72b692af29b1f220b2e04061eec9202811/tools/toolbox.py#L29 and https://github.com/libratbag/libratbag/blob/f82c6f72b692af29b1f220b2e04061eec9202811/tools/toolbox.py#L110-L112 import other modules...